fluxc/src/main/java/org/wordpress/android/fluxc/network/rest/wpcom/jetpacktunnel/JetpackTunnelGsonRequest.kt

```diff
@@ -240,36 +241,28 @@ object JetpackTunnelGsonRequest {
     private fun createTunnelParams(params: Map<String, String>, path: String): MutableMap<String, String> {
         val finalParams = mutableMapOf<String, String>()
         with(finalParams) {
-            put("path", buildRestApiPath(path, params, "get"))
+            put("path", "$path&_method=get")
             put("json", "true")
```

Comment:
also nit, but for consistency with `put("json", "true")`, can we split to

```suggestion
            put("path", "$path")
            put("_method", "get")
            put("json", "true")
```

?

Reply:
I don't think this will work, as the `_method` has to be part of the parameter `path`, and not a separate parameter, URL encoding will encode the `&` symbol, which makes sure that it's part of the parameter `path`.

Reply:
Thanks, I think got it!